src/Kunstmaan/FormBundle/Entity/FormSubmissionFieldTypes/FileFormSubmissionField.php

```diff
@@ -79,10 +80,8 @@ public function isNull()
 
     /**
      * Move the file to the given uploadDir and save the filename
-     *
-     * @param string $uploadDir
      */
-    public function upload($uploadDir, $webDir)
+    public function upload($webDir, Filesystem $fileSystem)
```

Comment:
This is actually a BC break, so the filesystem parameter should be optional an the old behavior should be deprecated and executed in that case I think

Reply:
Took care of it. Hope it's ok now.